lib/tasks/orcid.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
is the idea that we would have two separate schedules, one to update our author's table with ORCIDs that come from MAIS and one to push the works for all of those users?  both actions start with this same API call to MAIS right?

`Mais.client.fetch_orcid_users`

Reply:
They are two completely independent tasks. However, both tasks start with retrieving orcid info from MAIS.

Reply:
Understood, though they could in theory be done at the same time (update users and push to ORCID as you go), since both tasks will be happening on a schedule and one is dependent on the other.

Reply:
They're both independent. Pushing the works to Orcid does NOT used the orcid id stored in the db. Rather, it uses the orcid id and access token returned from MAIS.